net-dns/knot-resolver/files/kresd.initd

```diff
@@ -17,7 +26,6 @@ depend() {
     provide dns
 }
 
-start() {
-	checkpath --directory /var/run/kresd
-	default_start
+start_pre() {
+        checkpath -d -m 0750 -o "${KRESD_USER}:${KRESD_GROUP}" ${KRESD_RUNDIR}
```

Comment:
Isn't this applying the anti-pattern mentioned in https://github.com/OpenRC/openrc/blob/master/service-script-guide.md#pid-files-should-be-writable-only-by-root?